client/js/components/statement/assessmentTable/Fragment.vue

```diff
@@ -542,14 +538,32 @@ export default {
       return Base64.encode(this.fragment.text)
     },
 
-    selectedElementParagraph () {
-      return this.paragraph[this.fragment.elementId] || []
+    isClaimed () {
+      /*
+      * Some projects do not have the claim opportunity
+      * if the permission 'feature_statement_assignment' is activated,
+      * a statement or fragment is claimed if an assignee exist
+      * and the assignee should be the logged user so that a statement or fragment(='Datensatz') is claimed
+      * after claiming it the statement or fragment is editable for this user
+      * if this permission is not activated, the statement or fragment looks like already is claimed,
+      * so that any user can edit this part without claim because in the projects without this permission,
+      * there is no other opportunity that different user could have claimed this statement or fragment
+      */
```

Comment:
I don't find the current comment very easy to understand, maybe we could shorten it a little?
We could also consider renaming `isClaimed` to `isEditable`, since the fragment is not actually claimed if the claim feature doesn't exist

```suggestion
      /*
      * The fragment is only editable if 
      * a) in case the permission for the claim feature is active:
      * - the fragment has an assignee
      * - the assignee is the currently logged-in user
      * b) in case the permission for the claim feature is not active:
      * always
      */
```